ts-src/bip32.ts

```diff
@@ -0,0 +1,382 @@
+import * as crypto from './crypto';
+const bs58check = require('bs58check');
+const ecc = require('tiny-secp256k1');
+const typeforce = require('typeforce');
+const wif = require('wif');
+
+interface Network {
+  wif: number;
+  bip32: {
+    public: number;
+    private: number;
+  };
+  messagePrefix?: string;
+  bech32?: string;
+  pubKeyHash?: number;
+  scriptHash?: number;
+}
+
+const UINT256_TYPE = typeforce.BufferN(32);
+const NETWORK_TYPE = typeforce.compile({
+  wif: typeforce.UInt8,
+  bip32: {
+    public: typeforce.UInt32,
+    private: typeforce.UInt32,
+  },
+});
+
+const BITCOIN = {
```

Comment:
You might want to declare like this so you don't need to append `as Network` every time.
```
const BITCOIN: Network = {
```

Reply:
Ok, done. I also changed some as Buffers to use ! instead.